submissions/oluwaboriShobayo/React/to-do-app/src/Components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Footer = ({todos, setTodos}) => {
+    return (
+        <div className="footer">
+            <span className="item1">{todos.filter(todo => todo.completed === false).length} Items left</span>
```

Comment:
I think you should use a more descriptive class name like items-left, all-items, active-items etc @OluwaboriShobayo